src/renderer/pages/ConnectionView.tsx

```diff
@@ -77,21 +105,77 @@ const ConnectionView = (): JSX.Element => {
     ipcRenderer.send(VIEW_CONNECTION_LIST);
   };
 
+  const exportLogs = () => {
+    const blob = new Blob([JSON.stringify(filteredLogs)], {
+      type: 'application/json',
+    });
+    const link = document.createElement('a');
+    link.href = window.URL.createObjectURL(blob);
+    link.download = 'logs.json';
+    link.click();
+  };
+
+  const formatLog = (
+    msg: ConnectionStatusUpdate,
+    remoteAddr: string
+  ): SimplifiedLog => {
+    const date = new Date().toISOString();
+    const status = msg.lastError ? 'error' : 'info';
+    let message = '';
+
+    switch (msg.status) {
+      case 1:
+        message = msg.peerAddr + ' opening connection to ' + remoteAddr;
+        break;
+      case 2:
+        message =
+          msg.peerAddr +
+          ' authentication with ' +
+          msg.authUrl +
+          ' required for ' +
+          connection.remoteAddr;
```

Comment:
```suggestion
          remoteAddr;
```

as there's no `remoteAddr` property for `connection` and it causes `undefined`

<img width="916" alt="image" src="https://user-images.githubusercontent.com/673131/145106825-fd7dc71c-c380-4fbb-9a4e-531b0cb1815f.png">

Reply:
yeah missed this obe when i refactored